ibm_catalog.json

```diff
@@ -62,7 +62,7 @@
               ],
               "id": "95fccffc-ae3b-42df-b6d9-80be5914d852-global",
               "name": "deploy-arch-ibm-slz-ocp",
-              "version": ">=v3.0.0",
+              "version": ">=v3.0.0,<6.0.0",
```

Comment:
@NatarajBTI have you tried with v6.0.1 or later of landing zone? It should fix the issue you saw with storage. That means you can remove this upper constraint.